src/05-derivatives/03-imaging.md

```diff
@@ -0,0 +1,405 @@
+| Density      | REQUIRED if `den` is present. String, or [object][] mapping labels to strings. Specifies the interpretation of the density keyword.    |
+
+### Discrete Segmentations
+
+Discrete segmentations of brain tissue represent multiple anatomical structures
+(such as tissue class or Brodmann area) with a unique integer label in a 3D volume.
+See [Common image-derived labels](#common-image-derived-labels) for interpretation
+how integer values map to anatomical structures.
+
+Template:
+
+```Text
+<pipeline_name>/
+    sub-<participant_label>/
+        anat|func|dwi/
+            <source_entities>[_space-<space>][_res-<label>][_den-<label>]_dseg.nii.gz
+```
+
+Example:
+
+```Text
+pipeline/
+    sub-001/
+        anat/
+            sub-001_space-orig_dseg.nii.gz
+            sub-001_space-orig_dseg.json
+```
+
+A segmentation can be used to generate a binary mask that functions as a
+discrete "label" for a single structure.
+In this case, the mask suffix MUST be used, and the `label` entity SHOULD be used
+to specify the masked structure
+(see [Common image-derived labels](#common-image-derived-labels)),
+and the `Atlas` metadata SHOULD be defined.
+For example:
+
+```Text
+pipeline/
+    sub-001/
+        anat/
+            sub-001_space-orig_label-GM_mask.nii.gz
+```
+
+### Probabilistic Segmentations
+
+Probabilistic segmentations of brain tissue represent a single anatomical
+structure with values ranging from 0 to 1 in individual 3D volumes or across
+multiple frames.
+If a single structure is included the `label` entity SHOULD be used to specify
+the structure.
+
+Template:
+
+```Text
+<pipeline_name>/
+    sub-<participant_label>/
+        func|anat|dwi/
+            <source_entities>[_space-<space>][_res-<label>][_den-<label>][_label-<label>]_probseg.nii.gz
+```
+
+Example:
+
+```Text
+pipeline/
+    sub-001/
+        anat/
+            sub-001_space-orig_label-BG_probseg.nii.gz
+            sub-001_space-orig_label-WM_probseg.nii.gz
+```
+
+See [Common image-derived labels](#common-image-derived-labels)
+for reserved key values for `label`.
+
+A 4D probabilistic segmentation, in which each frame corresponds to a different
+tissue class, must provide a label mapping in its JSON sidecar. For example:
+
+```Text
+pipeline/
+    sub-001/
+	    anat/
+		    sub-001_space-orig_probseg.nii.gz
+		    sub-001_space-orig_probseg.json
+```
+
+The JSON sidecar must include the label-map key that specifies a tissue label
+for each volume:
+
+```JSON
+{
+    "LabelMap": [
+        "BG",
+        "WM",
+        "GM"
+        ]
+}
+```
+
+Values of `label` SHOULD correspond to abbreviations defined in
+[Common image-derived labels](#common-image-derived-labels).
+
+### Discrete surface segmentations
+
+Discrete surface segmentations (sometimes called *parcellations*) of cortical
+structures MUST be stored as GIFTI label files, with extension `.label.gii`.
```

Comment:
For consistency with other suggestion.

```suggestion
structures MUST be stored as GIFTI label files, with the extension `.label.gii`.
```

Reply:
I have one comment about segmentation/parcellation. The term segmentation is also used for White Matter tracts. These are tractography files with clustered streamlines into groups called white matter tracts. See an example here and image below:
https://brainlife.io/pub/5ce2c9ed40363b00353677d4#5b0dbdcfe81b47020407ca17

The word parcellation is not used for white matter tracts. Currently, segmentation is used: https://bids-specification.readthedocs.io/en/common-derivatives/05-derivatives/03-imaging.html#segmentations

If it were to be possible to keep parcellation for voxels and surfaces, it would be easier later to adopt the term segmentation for the white matter

![image](https://user-images.githubusercontent.com/2119795/83689356-e608ea00-a5bc-11ea-940f-4bd904f4cbf3.png)

@arokem @oesteban @Lestropie

Reply:
Thanks for the comment, @francopestilli!

I don't necessarily see this as being in conflict. The concept of segmentation as a labeling of anatomical structures is broad enough to include voxels, vertices and tracts, so this shouldn't preclude the use of `dseg` and `probseg` in tractography.

I also don''t think NIfTI/GIFTI/CIFTI are used to represent white matter tracts, so there would not be any confusion as to what kind of segmentation you're dealing with, either. For example, you could look for surface segmentations with a query like `layout.get(suffix='dseg', extension='.gii')`, and a tractography segmentation with a query like `layout.get(suffix='dseg', extension='.trk')` (or whatever preferred file type BEP016 uses).

Does this address your concerns? If not, could you provide an example of how our current definitions or naming conventions will conflict with how you would prefer to name files?

Reply:
OK great. As long as the term is not going to be reserved for the use on voxels-based segmentations, then the rest will work out well! We can close this.